docs/api/rest_api/rest_api_reference/input/ez-types.raml

```diff
@@ -2128,401 +2140,1056 @@ View: #TODO outdated example
+            type: mixed
+        defaultAlwaysAvailable:
+            type: boolean
+        names:
+            type: array
+        descriptions:
+            type: array
+        modifierId:
+            description: 'If set, this value overrides the current user as creator.'
+            type: mixed
+        modificationDate:
+            type: DateTime
 
 ViewInput:
+    description: 'This class represents a View input.'
     type: BaseObject
+    properties:
+        identifier:
+            description: 'Content identifier.'
+            type: string
+        FilterQuery:
+            type: object
 
 UserCreate:
+    description: 'This class is used to create a User.'
     type: BaseObject
+    properties:
+        mainLanguageCode:
+            type: string
+        remoteId:
+            type: string
+        login:
+            type: string
+        email:
+            type: string
+        password:
+            type: string
+        enabled:
+            type: boolean
 
 UserGroupCreate:
+    description: 'This class is used to create a User Group.'
     type: BaseObject
+    properties:
+        mainLanguageCode:
+            type: string
+        remoteId:
+            type: string
+        fields:
+            type: Field[]
 
 RoleAssignInput:
+    description: 'This class represents a Role assign input.'
     type: BaseObject
+    properties:
+        id:
+            description: 'The unique ID of the Role assignment.'
+            type: integer
+        Role:
+            description: 'Returns the Role to which the user or user group is assigned to.'
+            type: Role
+        Limitation:
+            description: 'Returns the Limitation of the Role assignment.'
+            type: Limitation
 
 UserUpdate:
+    description: 'This class is used to update a User.'
     type: BaseObject
+    properties:
+        login:
+            type: string
+        email:
+            type: string
+        password:
+            type: string
+        enabled:
+            type: boolean
+        maxLogin:
+            type: integer
+        ContentUpdate:
+            description: 'The update structure for the profile content.'
+            type: ContentUpdate
+        ContentMetadataUpdate:
+            description: 'The update structure for the profile metadata.'
+            type: ContentMetadataUpdate
 
 RoleInput:
+    description: 'This class represents a Role input.'
     type: BaseObject
+    properties:
+        identifier:
+            type: string
 
 PolicyCreate:
+    description: 'This class is used to create a Policy.'
     type: BaseObject
+    properties:
+        module:
+            description: 'Name of module associated with the Policy. For example, content.'
+            type: string
+        function:
+            description: 'Name of the module function, or all functions with '*'. For example, read.'
```

Comment:
`Unexpected characters near "*'. For example, read.'" at line 2311 (near "description: 'Name of the module function, or all functions with '*'. For example, read.'").`

In YAML (and RAML by extension), to escape a single quote, just double it:

```suggestion
            description: 'Name of the module function, or all functions with ''*''. For example, read.'
```